oneops-admin/lib/shared/exec-gems-az.yaml

```diff
@@ -4,7 +4,7 @@ common:
       - 1.29.0
     -
       - fog-core
```

Comment:
@abbas-sheikh-confiz reason fog-core was added to `common` was because `fog-azure-rm 0.3.5` has hard dependency on `1.4.30`. since the newer version of `fog-azure-rm` took out this dependency and depends on `~>1.43` we can take out `fog-core` from `common`. please remove it, validate and update the PR